plugins.go

```diff
@@ -91,6 +91,10 @@ func (p *Plugins) Retrieve() (err error) {
 			continue
 		}
 
+		if len(pi.gitURL) >= 1 && pi.gitURL[0] == '.' {
+			continue
```

Comment:
Can we add a comment explaining why we're continuing?

Reply:
Agreed. "Skip update for relative paths" makes sense.

But then, are there any times were we would want to update a relative path with vpm? Naw, probably don't need to support it, or if we do, we can make it a flag in the future.

Reply:
do we want to include the full `./` check in relative paths? Or just consider `.` sufficient?